python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -264,6 +265,30 @@ def find_never_materialized_or_requested_root_asset_partitions(
     )
 
 
+def can_materialize_candidates_unit(
```

Comment:
Could this have a more specific name that refers to the fact that it's filtering based on time?